openssl/src/aes.rs

```diff
@@ -258,6 +258,36 @@ pub fn unwrap_key(
     }
 }
 
+/// Encrypts an aes-128/256 block from `in` to `out`.
+/// * `key`: The key-encrypting-key to decrypt the wrapped key. Must be a decrypting key
+/// * `out`: The buffer to write the plaintext to
+/// * `in_`: The input plaintext
+#[corresponds(AES_encrypt)]
+pub fn aes_encrypt(key: &AesKey, out: &mut [u8], in_: &[u8]) {
```

Comment:
How big do these buffers need to be? What happens if they're not big enough?

Reply:
specified via types